public/components/security/roles-mapping/components/roles-mapping-table.tsx

```diff
@@ -58,10 +60,23 @@ export const RolesMappingTable = ({ rolesEquivalences, rules, loading, editRule,
     {
       field: 'id',
       name: 'Status',
-      render: item => {
-        return WzAPIUtils.isReservedID(item) && <EuiBadge color="primary">Reserved</EuiBadge>;
+      render (item, obj){
+        if(WzAPIUtils.isReservedID(item)){
+          if( (obj.name == "wui_elastic_admin" || obj.name == "wui_opendistro_admin")){
```

Comment:
always when it is possible, try to compare with numbers instead of strings, the id should be the field to compare
and use constants, to avoid avoid keeping values in different places in code

Reply:
Okay, so I proceed to change it and will keep in mind for future occasions. Thank you!